plugin.yaml

```diff
@@ -2,7 +2,6 @@ plugins:
 
   aws:
     executor: central_deployment_agent
-    source: https://github.com/cloudify-cosmo/cloudify-aws-plugin/archive/2.4.4.zip
```

Comment:
@kfrukacz-gs why did you drpp the source ?
and more thing we need to increment the 2.4.4 to 2.4.5 and also check the setup.py as well

Reply:
ok

Reply:
done